kuma/payments/jinja2/payments/includes/payments-banner.html

```diff
@@ -53,26 +56,33 @@ <h4 tabindex="-1">{{_('Support MDN, and it comes back to you.')}}</h4>
                             <a href="{{ payments_url }}" {% if not recurring_payment %} class="active" {% endif %}>
                                 {{_('One off payment')}}
                             </a>
-                            <a href="{{ recurring_payment_subscription_url }}" {% if recurring_payment %} class="active" {% endif %}>
+                            <a href="{{ recurring_payment_initial_url }}" {% if recurring_payment %} class="active" {% endif %}>
                                 {{_('Monthly payment')}}
                             </a>
                         </div>
-                        {# One Time payments form #}
-                        {% with form=form, payments_url=payments_url %}
-                            {%- include "payments/includes/payments-form.html" %}
-                        {% endwith %}
+
                         {% if recurring_payment %}
-                            {% if user.username %}
+                            {% if user.is_authenticated %}
                                 {% with form=form, is_popover=False, recurring_payment=recurring_payment, payments_url=payments_url, recurring_payment_subscription_url=recurring_payment_subscription_url  %}
-                                    {#- include "payments/includes/payments-form.html" #}
+                                    {%  include "payments/includes/payments-form.html" %}
                                 {% endwith %}
                             {% else %}
-                                {% set github_url = provider_login_url('github', next=next_url) %}
-                                <p>{{_('We’ve noticed you’re not logged in to MDN. To complete your monthly payment, we’ll need you to log in first.')}}</p>
-                                <a href="{{ github_url }}" class="payment-form-button" data-service="GitHub" rel="nofollow">
-                                    {{ _('Sign in') }}{% include 'includes/icons/social/github.svg' %}
-                                </a>
+                                {% with form=form, hide_submit=True, is_popover=False, recurring_payment=recurring_payment, payments_url=payments_url, recurring_payment_subscription_url=recurring_payment_subscription_url  %}
```

Comment:
@joshJarr What is `hide_submit=True` used for? I do not see it being used anywhere else other than this line.

Reply:
Addressed, great spot, I think this was leftover from a previous implementation. Thanks!